org.zim_wiki.Zim.json

```diff
@@ -2,7 +2,7 @@
     "app-id": "org.zim_wiki.Zim",
     "branch": "stable",
     "runtime": "org.gnome.Platform",
-    "runtime-version": "41",
+    "runtime-version": "42",
```

Comment:
The "version" for the texlive extension should have been updated to 21.08 when the runtime was bumped to 41. And it should be done (42 also uses 21.08)

Reply:
Done